tracks/python/exercises/phone-number/mentoring.md

```diff
@@ -0,0 +1,54 @@
+# Mentoring
+
+## Phone Number
+
+The problem asks you to write a `PhoneNumber` class.
+The constructor takes an input string that may or may not contain a valid number.
+The constructor needs to validate the number.
+The `pretty()` method needs to return a formatted number.
+
+## Reasonable Solution
+
+```python
+class Phone:
+    """Phone number validator."""
+
+    def __init__(self, number: str):
```

Comment:
As a reasonable solution, it looks good, but I would not use `number` if not asking for a number.  `phoneNumber` hints enough that it is not strictly a number, but likely a number represented as a string.

Still, probably no reason to change, will probably come up if names come up as the way we communicate to humans using the code.

Reply:
This variable name is set in the stub. I'd prefer to match the signature given in the stub.

Reply:
Definitely. Some things are "just given" but we are writing notes, not changing the code.  So it could be mentioned, even if we are not to do anything with it.